awx/ui/client/lib/components/list/list-toolbar.partial.html

```diff
@@ -1,5 +1,24 @@
 <div class="at-List-toolbar--attached">
-  <div ng-class="isCollapsed ? 'active' : ''" ng-if="!sortOnly" class="at-List-toolbar-item" ng-click="onCollapse()">Compact</div>
-  <div ng-class="!isCollapsed ? 'active' : ''" ng-if="!sortOnly" class="at-List-toolbar-item" ng-click="onExpand()">Expanded</div>
-  <div class="at-List-toolbar-item">Sort</div>
+    <div ng-class="isCollapsed ? 'active' : ''" ng-if="!sortOnly" class="at-List-toolbar-item" ng-click="onCollapse()">{{ vm.strings.get('toolbar.COMPACT') }}</div>
+    <div ng-class="!isCollapsed ? 'active' : ''" ng-if="!sortOnly" class="at-List-toolbar-item" ng-click="onExpand()">{{ vm.strings.get('toolbar.EXPANDED') }}</div>
+    <div ng-if="sortOptions" class="at-List-toolbarDropdown dropdown">
```

Comment:
https://github.com/ansible/awx/pull/3356#issuecomment-469387019
`ng-if="sortOptions"` will hide the Sort dropdown when sortOptions are not defined.